init/pkg/update_pod_status_util.go

```diff
@@ -137,50 +131,50 @@ func getPodImage(k8sClient client.Client, ctx goctx.Context, podNamespacedName t
 }
 
 //nolint:unparam // generic function
-func getEnv(key, fallback string) string {
+func getEnv(key string) string {
 	if value, ok := os.LookupEnv(key); ok {
 		return value
 	}
 
-	return fallback
+	return ""
```

Comment:
We can remove this func all together and use os.Getenv()

Reply:
there are certain flags which needs to be available in env else system should panic, Hence adding panic if those flags not found.